stable/nginx-ingress/values.yaml

```diff
@@ -65,6 +65,11 @@ controller:
   #    maxUnavailable: 1
   #  type: RollingUpdate
 
+  # minReadySeconds to avoid killing pods before we are ready
+  ##
+  minReadySeconds: 15
```

Comment:
Isn't that kind of high as a default?

Reply:
We've been using this in production for a while - it may be ok to lower it. What would you suggest?

Reply:
The default is zero. I think we should stick to that.

Reply:
Fixed 👍